packages/core/src/payment/strategies/paypal-commerce/paypal-commerce-request-sender.ts

```diff
@@ -67,4 +67,18 @@ export default class PaypalCommerceRequestSender {
 
         return res.body;
     }
+
+    async updateOrder(payload: UpdateOrderPayload) {
+        const url = `/api/storefront/initialization/paypalcommerce`;
```

Comment:
Naming seems a bit off here since the method is called `updateOrder` where as the url called has nothing related to update :|

Reply:
Yeah... I know that the url has nothing related to update. But we use `put` method with `request sender` to update paypal order on BE side